lib/playground.dart

```diff
@@ -278,17 +281,24 @@ class Playground implements GistContainer, GistController {
     _editpanel.children.first.attributes['flex'] = '';
     editor.resize();
 
-    keys.bind(['ctrl-s'], _handleSave);
-    keys.bind(['ctrl-enter'], _handleRun);
+    keys.bind(['ctrl-s'], sharingDialog.show, "Share");
```

Comment:
Let's keep this bound to the no-op 'save' call. User's have muscle memory of what hitting ctrl-s will do; they would have to un-learn it for dartpad.